src/neo-vm/InteropService.cs

```diff
@@ -17,14 +23,16 @@ public InteropService()
 
         protected void Register(string method, Func<ExecutionEngine, bool> handler)
         {
-            dictionary[method.ToInteropMethodHash()] = handler;
+            uint hash = method.ToInteropMethodHash();
+            dictionary[hash] = handler;
+            dictionaryStr[method] = hash;
         }
 
         internal bool Invoke(byte[] method, ExecutionEngine engine)
         {
             uint hash = method.Length == 4
                 ? BitConverter.ToUInt32(method, 0)
-                : method.ToInteropMethodHash();
+                : dictionaryStr[method];
```

Comment:
Here we should choose in wich dictionary we need to looking for, otherwise we can get exceptions too